jobs/sql/extractions/all_encounters_staging.sql

```diff
@@ -16,27 +16,29 @@ encounter_type_name varchar(50),
 entered_datetime    datetime,
 emr_id              varchar(15),
 next_appt_date      date,
-index_asc			int,
-index_desc			int
+voided				bit
 );
 
+select last_loaded_time into @lastloadtime from petl_load_times where loaded_domain = 'encounter';
+
 Insert into temp_all_encounters (
-encounter_id, encounter_datetime, encounter_type_name, encounter_location, patient_id, visit_id, user_entered, entered_datetime)
-select encounter_id, encounter_datetime, encounter_type_name_from_id(encounter_type), location_name(location_id), patient_id, visit_id, person_name_of_user(creator), date_created 
+encounter_id, encounter_datetime, encounter_type_name, encounter_location, patient_id, visit_id, user_entered, entered_datetime, voided)
+select encounter_id, encounter_datetime, encounter_type_name_from_id(encounter_type), location_name(location_id), patient_id, visit_id, person_name_of_user(creator), date_created, voided 
 from encounter 
-where voided = 0
+where (date_created > @lastloadtime or 
+	date_changed > @lastloadtime or 
```

Comment:
Not sure if you need to check date_voided here as well (or if date_changed will cover it) but it couldn't hurt...

Reply:
yeah - was thinking the same thing.  WIll do